autotest/gdrivers/tiledb_write.py

```diff
@@ -86,6 +88,69 @@ def test_tiledb_write_rgb():
 
     gdaltest.tiledb_drv.Delete('tmp/tiledb_rgb')    
 
+@pytest.mark.require_driver('TileDB')
+def test_tiledb_write_attributes():
+    try:
+        import ctypes
+    except ImportError:
+        pytest.skip()
+
+    for libname in ['msvcrt', 'libc.so.6']:
```

Comment:
aouch ! You don't really need to go through such low level stuff to create a in-memory dataset that can be reference by a name.
```src_ds = gdal.GetDriverByName('GTiff').Create('/vsimem/temp.tif', w, h, num_bands, gdal.GDT_Int32)``` will do the job in a more compact, safer and platform independent way.
Clean up with gdal.GetDriverByName('GTiff').Delete('/vsimem/temp.tif')

Reply:
:+1: